src/UI/templates/default/ViewControl/viewcontrol.less

```diff
@@ -0,0 +1,20 @@
+.il-viewcontrol-section .glyphicon:before {
+  background-color: lighten(@brand-primary, 15%);
+  border: 1px solid transparent;
+  padding: 3px 8px;
+  position: relative;
+  top: 2px;
```

Comment:
There is still something a little off in the positioning. The button in the middle is a little further down than the others.